scripts/winesapos_convert.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
```

Comment:
For consistency with other scripts, this should be renamed from `winesapos_convert.sh` to `winesapos-convert.sh` (using a dash `-` instead of an underscore `_`).

Also make sure this file is executable before committing.

```
mv scripts/winesapos_convert.sh scripts/winesapos-convert.sh
chmod +x scripts/winesapos-convert.sh
git add scripts/winesapos-convert.sh
git commit -m "[convert] Rename script and make executable"
```